docs/large_inputs.md

```diff
@@ -1,149 +1,172 @@
 # Handling large inputs
 
-The Variant Transforms pipeline can process hunderds of thousands of files,
-millions of samples, and billions of records. There are a few settings that
-may need to be adjusted depending on the size of the input files. Each of these
-settings are explained in the sections below.
+The Variant Transforms pipeline can process hundreds of thousands of files,
+millions of samples, and billions of records. There are a few settings that 
+may need to be adjusted depending on the size of the input files. Each of 
+these settings are explained in the sections below.
 
-Default settings:
+Example usage:
```

Comment:
Can we drop the flags that have sensible defaults ?

And just show what flags they need to provide?